gradle.properties

```diff
@@ -3,7 +3,7 @@ version=1.2.0
 gsonVersion=2.8.6
 junitVersion=5.6.2
 mockitoVersion=3.3.3
-newRelicAgentVersion=6.1.0
+newRelicAgentVersion=6.4.2
```

Comment:
The version here should match the api version in the java agent. It can lead to issues if they do not match.

Reply:
So does this mean that we need to publish a version of jfr for each new version of the agent?

Reply:
Only when there are changes to the Java agent API.

Reply:
I updated this variable to be `newRelicAgentApiVersion` to be clearer as we don't actually use the agent itself in this project at all.